### DISCUSSION TITLE
Dark mode on Firefox needs to be activated for every single page

### DISCUSSION THREAD

**Discussion by LuizFarias** [squidfunk/mkdocs-material#3569] [Q&A] [answered]
Hello. I have been using the material to write a manual, and we realized with the build of the generated site that we need to activate the dark mode for each page in Firefox.

Has anyone gone through this issue and would know if it is a problem with the browser itself and I need to apply customization for Firefox? In other browsers, we can activate it only once and the theme is applied to everything.

- mkdocs  version 1.2.3
- mkdocs-material  version 8.1.7
- Firefox version 96
- System: Linux, Windows

**Comment by squidfunk** [comment, marked as answer]
Dark mode settings are persisted in local storage. Do you have local storage disabled?

**Reply by LuizFarias** [reply]
I have the dom.storage.enabled config set to true. I can see that when I set it to false, the light/dark switch button disappears.
It happens when I open the generated HTML directly on Firefox. When I use mkdocs serve, the dark theme persists on all pages.

**Reply by squidfunk** [reply]
Hmm, interesting. There's something behaving differently in Webkit and Firefox when it comes to identifying the prefix of the generated site. Please create an issue, I can take a look we I find some time.

**Reply by LuizFarias** [reply]
I'll do that. Thank you for answering.
